axiom/monitors.go

```diff
@@ -47,6 +54,14 @@ func operatorFromString(s string) (c Operator, err error) {
 	return c, err
 }
 
+func typeFromString(s string) (c Type) {
+	switch s {
+	case MatchEvent.String():
+		return MatchEvent
+	}
```

Comment:
is this not missing the threshold type?

Reply:
It was defaulting to Threshold. Have refactored